packages/dds/ink/README.md

```diff
@@ -23,12 +23,12 @@ library consumers should always prefer `^`.
 To get started, install the package by running the following command:
 
 ```bash
-npm i @fluidframework/ink
+npm i @fluid-experimental/ink
 ```
 
 ## API Documentation
 
-API documentation for **@fluidframework/ink** is available at <https://fluidframework.com/docs/apis/ink>.
+API documentation for **@fluid-experimental/ink** is available at <https://fluidframework.com/docs/apis/ink>.
```

Comment:
While you are updating this document, could you also confirm if the api documentation link is working for you. For me, this link only works when I go to the v2 docs

https://fluidframework.com/docs/api/v2/ink

Reply:
Most likely this is AB#7096